src/sources/kubernetes_logs/pod_metadata_annotator.rs

```diff
@@ -20,6 +21,7 @@ pub struct FieldsSpec {
     pub pod_name: String,
     pub pod_namespace: String,
     pub pod_uid: String,
+    pub pod_ips: String,
```

Comment:
I think we should allow assigning both `pod_ip` and `pod_ips` fields in the event. See my other comments for rationale.